fab_oidc/views.py

```diff
@@ -1,10 +1,14 @@
+import os
 from flask import redirect, request
 from flask_appbuilder.security.views import AuthOIDView
 from flask_login import login_user
 from flask_admin import expose
 from urllib.parse import quote
 
 
+# Set the OIDC field that should be used as a username
+USERNAME_OIDC_FIELD = os.getenv('CONF_USERNAME_OIDC_FIELD', default='sub')
```

Comment:
this should be `USERNAME_OIDC_FIELD` to match the README?

Reply:
Good catch, I altered it 👍